packages/core/tsconfig.json

```diff
@@ -5,6 +5,5 @@
     "rootDir": ".",
     "baseUrl": "."
   },
-  "references": [{ "path": "../sovran" }],
```

Comment:
Looks like `tsconfig.linter.json` also has this reference

Reply:
Good catch!